app/scripts/modules/kubernetes/instance/details/details.controller.js

```diff
@@ -29,6 +29,10 @@ module.exports = angular.module('spinnaker.instance.detail.kubernetes.controller
       standalone: app.isStandalone,
     };
 
+    this.uiLink = function uiLink() {
```

Comment:
Can `account`, `region`, or `name` change on a running instance? (i.e., is there a reason to set `this.uiLink` to a function instead of just a string?)